server/src/ws.rs

```diff
@@ -0,0 +1,123 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use axum::extract::ws::{Message, WebSocket};
+use axum::extract::{Query, State, WebSocketUpgrade};
+use futures_util::stream::SplitSink;
+use futures_util::{SinkExt, StreamExt};
+use rustc_hash::FxHashMap;
+use serde::{Deserialize, Serialize};
+use tokio::sync::{mpsc, Mutex};
+use ton_block::{Deserializable, HashmapAugType, MsgAddressInt};
+use ton_types::{AccountId, HashmapType};
+
+use crate::server::Server;
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct WebSocketUpgradeQuery {
+    client_id: uuid::Uuid,
+}
+
+pub async fn ws_router(
+    ws: WebSocketUpgrade,
+    State(ctx): State<Arc<Server>>,
+    Query(query): Query<WebSocketUpgradeQuery>,
+) -> axum::response::Response {
+    ws.on_upgrade(move |socket| handle_socket(query.client_id, ctx, socket))
+}
+
+async fn handle_socket(client_id: uuid::Uuid, state: Arc<Server>, socket: WebSocket) {
+    let (tx, rx) = mpsc::unbounded_channel();
```

Comment:
Can you please use `channel(100)` with [send_timeout](https://docs.rs/tokio/latest/tokio/sync/mpsc/struct.Sender.html#method.send_timeout) instead?